packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -99,32 +99,34 @@ export class Builder extends EventEmitter {
   once(event: 'error', handler: (error: Error) => void): this;
   once(event: 'build:docs', handler: (built: DocumentBuild) => void): this;
   once(event: 'build:schema', handler: (built: SchemaBuild) => void): this;
-  once(event: 'start:docs', handler: () => void): this;
-  once(event: 'end:docs', handler: () => void): this;
-  once(event: 'start:schema', handler: () => void): this;
-  once(event: 'end:schema', handler: () => void): this;
+  once(
+    event: 'start:docs' | 'end:docs' | 'start:schema' | 'end:schema',
+    handler: () => void,
+  ): this;
```

Comment:
Neat, a rule does this too?

Reply:
I know right? https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/unified-signatures.md